test/actions/katello/upstream_subscriptions/remove_entitlements_test.rb

```diff
@@ -0,0 +1,63 @@
+require 'katello_test_helper'
+
+describe ::Actions::Katello::UpstreamSubscriptions::RemoveEntitlement do
```

Comment:
@akofink I'm testing that action here - do you want me to put this in a separate file?

Reply:
Oh! lol Yeah, that'd be good. :) Just for the reason that people looking for these tests will look there (including myself just now).